core/options.cpp

```diff
@@ -197,18 +184,42 @@ bool Options::Parse(int argc, char *argv[])
 		return false;
 	}
 
+	if (hdr != "off" && hdr != "single-exp" && hdr != "sensor" && hdr != "auto")
+		throw std::runtime_error("Invalid HDR option provided: " + hdr);
+
+	if (!verbose || list_cameras)
+		libcamera::logSetTarget(libcamera::LoggingTargetNone);
+
+	// HDR control. Set the sensor control before opening or listing any cameras.
+	// Start by disabling HDR unconditionally. Reset the camera manager if we have
+	// actually switched the value of the control.
+	set_subdev_hdr_ctrl(0);
+	app_->initCameraManager();
+
+	// Unconditionally disable libcamera logging for a bit.
+	libcamera::logSetTarget(libcamera::LoggingTargetNone);
```

Comment:
Does this suppress only infos and debugs, or does it remove errors and warnings?
We might want to keep the latter.